<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgChoosePathAndPrefix</class>
 <widget class="QDialog" name="DlgChoosePathAndPrefix">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>223</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose Path &amp; Prefix</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>180</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout" columnstretch="1,3,2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>path</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lineEditPath"/>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButtonChoosePath">
      <property name="text">
       <string>choose...</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="toolTip">
       <string/>
      </property>
      <property name="text">
       <string>prefix</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEditPrefix">
      <property name="toolTip">
       <string>wavenames will be prefixed with this</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>68</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_pxp_export">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>281</width>
     <height>17</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>all traces are stored in one file</string>
   </property>
   <property name="text">
    <string>create pxp file</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_create_datafolders">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>321</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>create folder structure (only for pxp files)</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxLevel">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>120</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Group</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Series</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>last folder level:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DlgChoosePathAndPrefix</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DlgChoosePathAndPrefix</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
